src/main/java/my/bookstore/dto/order/OrderStatusRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package my.bookstore.dto.order;
+
+import lombok.Data;
+import my.bookstore.lib.OrderStatus;
+
+@Data
+public class OrderStatusRequestDto {
+    @OrderStatus
+    private String status;
```

Comment:
@OrderStatus is redundant and make your code more complex.

Just use Enum here. 
```suggestion
    private OrderStatusEnum status;
```